wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallGridViewController.swift

```diff
@@ -53,7 +53,7 @@ final class CallGridViewController: UIViewController {
         PinchToZoomRule(isOneToOneCall: configuration.callHasTwoParticipants)
     }
 
-    private var visibleClientsSharingVideo: [AVSClient] = []
+    private var visibleClientsSharingVideo: [AVSClientVideoStream] = []
```

Comment:
suggestion: store the set, so that the conversion from array to set is not done more often than required.
```suggestion
    private var visibleClientsSharingVideo: Set<AVSClientVideoStream> = []
```

Reply:
done